test/functional/apps/discover/group3/_request_counts.ts

```diff
@@ -144,7 +146,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         );
         await waitForLoadingToFinish();
         // creating the saved search
-        await expectSearches(type, 2, async () => {
+        await expectSearches(type, savedSearchesRequests ?? 2, async () => {
```

Comment:
ℹ️ I dont understand why it runs 4 times on a saved search, I am looking into it from yesterday but I am def something missing. I would love the help from the https://github.com/orgs/elastic/teams/kibana-data-discovery team as they know this code better than me. We can do it on a follow up PR as I would love to merge the feature branch as soon as possible as multiple PRs depend on it.

Reply:
had a quick look at it, without having a solution. Agreed that this can be done in a follow up PR by @elastic/kibana-data-discovery

Reply:
ideally we could  add  a TODO comment with a linked issue

Reply:
I will do that :) Thanx for checking!

Reply:
Done!